how_to_eurec4a/_config.yml

```
@@ -23,6 +23,6 @@ html:
   use_edit_page_button      : true
 execute:
   execute_notebooks         : force
-  timeout: 150
+  timeout: 180
```

The suggested fix to the flight tracks should significantly speed the notebook up which makes this change unnecessary.
In general, I'd like to keep the notebook execution times low (and probably we could even reduce the limit further than what it's currently set to).
```suggestion
  timeout: 150
```